src/components/Challenges.js

```diff
@@ -1,16 +1,108 @@
+                    We challenge you to build something that embodies our values of exploration and teamwork.
+                    This could involve a project to express your gratitude to nature or a survival tool for you and your fellow hackers.
+                  </p>
+                </div>
+              </Col>
+            </Row>
+          </div>
+        </Carousel.Item>
+
+        <Carousel.Item>
+          <div className="carouselItem">
+            <Row>
+              <Col className="carouselImage">
+                <OnionCorn id="OnionCorn"/>
+              </Col>
+              <Col>
+                <div className="carouselHeader">
+                  <h2>Most Inclusive Hack:</h2>
+                </div>
+                <div className="carouselCaption">
+                  <p>Last year was full of social change, and it doesn’t need to stop there! We challenge you to continue the trend into 2021 by creating a hack that uses tech to promote social good, accessibility, or impact on the world around you. <br/>
+                    - Did you know there <a href="https://www.w3.org/TR/wai-aria-1.1/">is a spec for web accessibility?</a> <br/>
+                    - Your code doesn’t need to be a user-facing application!  Checkout <a href="https://github.com/dequelabs/axe-core-npm/tree/develop/packages/react">this plugin that helps developers write accessible code.</a> <br/>
+                        - Want to build an app for social good, but need inspiration?  Checkout <a href="https://buildforblacklives.com/">Build4BlackLive </a>, built by some of our own members from HackBeanpot Core!<br/>
+                  </p>
+                </div>
+              </Col>
+            </Row>
+          </div>
+        </Carousel.Item>
+
+        <Carousel.Item>
+          <div className="carouselItem">
+            <Row>
+              <Col className="carouselImage">
+                <CelebratoryBoi id="CelebratoryBoi"/>
+              </Col>
+              <Col>
+                <div className="carouselHeader">
+                  <h2>Hack for Joy:</h2>
+                </div>
+                <div className="carouselCaption">
+                  <p>Everyone has their own unique hobbies and interests, such as gaming, rock climbing, knitting, or fishing.
+                    Whether it’s a hobby you picked up during quarantine or a life-long passion, at HackBeanpot, we challenge you to build whatever your heart desires!
+                    Encouraging passion and creativity, we present to you, Hack for Joy!
+                  </p>
+                </div>
+              </Col>
+            </Row>
+          </div>
+        </Carousel.Item>
+
+        <Carousel.Item>
+          <div className="carouselItem">
+            <Row>
+              <Col className="carouselImage">
+                <BeanstalkBoi id="BeanstalkBoi"/>
```

Comment:
not sure if this is a design thing but in the screenshot this seems cut off from the top

Reply:
I think it's supposed to be at the top so it looks like the stalk is trailing off, but it looks weird like that because the section needs to be a certain height to fit all of the other beans and text. I went ahead and just allowed it to move to the center on larger screens, but let me know if you think that looks to weird (will add screenshots in a sec).